### PR TITLE
71 implement shork

### PR DESCRIPTION
**Pull request by flamion**

### CODE REVIEW

File: backend/shork/src/main/kotlin/Settings.kt [hunk 1]
```diff
@@ -12,10 +12,22 @@ import software.shonk.interpreter.internal.instruction.Dat
  *   interpreter stops. -1 for no limit
  */
 class Settings(
-    val coreSize: Int,
-    val instructionLimit: Int,
-    val initialInstruction: String,
-    val maximumTicks: Int,
+    /** The size of the core, in number of instructions it will fit. */
+    val coreSize: Int = 8092,
```
Comment (PietHelzel) [inline backend/shork/src/main/kotlin/Settings.kt:16]: We should use a more common coresize. KOTH specifies a default coresize of 8000, the "official" standard uses 8192.

File: backend/shork/src/main/kotlin/Settings.kt [hunk 2]
```diff
@@ -12,10 +12,22 @@ import software.shonk.interpreter.internal.instruction.Dat
...
+    /**
+     * The maximum number of instructions will be executed before the interpreter stops and a draw
+     * is called.
+     */
+    val instructionLimit: Int = 100000,
+    /** The initial instruction the core will be filled with. */
+    val initialInstruction: String = "DAT #42, #1337",
```
Comment (PietHelzel) [inline backend/shork/src/main/kotlin/Settings.kt:23]: For compatibility, we should use the default "DAT $0, $0" instruction. Programs might use the a-field and b-field for operation, so it is a more suitable default instruction.

File: backend/shork/src/test/kotlin/TestInternalShork.kt [hunk 3]
```diff
@@ -42,27 +48,26 @@ internal class TestInternalShork {
 
     @Test
     fun `Player can win`() {
```
Comment (PietHelzel) [inline backend/shork/src/test/kotlin/TestInternalShork.kt:50]: Other test names are not capitalized, this one should start with a lowercase character too

**Review by PietHelzel — CHANGES_REQUESTED** [review]

File: backend/shork/src/main/kotlin/Settings.kt (hunk 1 above)
Comment (flamion) [reply backend/shork/src/main/kotlin/Settings.kt:16]: Yup, I forgot the 1 for 8192

**Review by PietHelzel — APPROVED** [review]

**Review by nikkischnelle — APPROVED** [review]
lgtm

**Pull request merged**